README.md

```diff
@@ -7,25 +7,38 @@
   Airflow Sample Provider
 </h1>
   <h3 align="center">
-  Guildelines on building, deploying, and maintaining provider packages that will help Airflow users interface with external systems. Maintained with ❤️ by Astronomer.
+  Guidelines on building, deploying, and maintaining provider packages that will help Airflow users interface with external systems. Maintained with ❤️ by Astronomer.
 </h3>
 
 <br/>
 
-This repository demonstrates best practices for building, structuring, and deploying Airflow provider packages as independent python modules available on PyPI.
+This repository provides best practices for building, structuring, and deploying Airflow provider packages as independent python modules available on PyPI.
 
-## Requirements
+Provider repositories must be public on Github and follow the structural and technical guidelines laid out in this Readme. Ensure that all of these requirements have been met before submitting a provider package for community review.
 
-Provider repositories must be public on Github and follow the structural and technical guidelines laid out in this Readme.
+Here, you'll find information on requirements and best practices for key aspects of your project:
 
-The package should be named as follows:
+- File formatting
```

Comment:
I always like to include a high level, table-of-contents-esque section so that readers know what the guide covers and what it doesn't.